.travis.yml

```diff
@@ -10,4 +10,10 @@ notifications:
     on_success: change  # options: [always|never|change] default: always
     on_failure: always  # options: [always|never|change] default: always
     on_start: false     # default: false
+env:
+  global:
+  - secure: STYk3FPnQwBHT25T.../O2SWbVYIL74K...XQAw=
+  - secure: MwRqcAgpBGrmyRY6.../lB/sjDhMk67Y...Xjgk=
 
+after_success:
+  - "[[ $TRAVIS_BRANCH == \"master\" ]] && { sbt publish; };"
```

Comment:
@dwijnand  thanks for spotting this :)

As luck would have it, @milessabin also updated [the shapeless script](https://github.com/milessabin/shapeless/blob/master/.travis.yml) earlier so I shall base the change on that and trust that this citation to his prior art is sufficient ;)